frontend/scripts/react-components/shared/hero.component.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import TwitterFeed from 'react-components/home/twitter-feed.component';
+
+function Hero(props) {
```

Comment:
Not really sure what a Hero is in this context, perhaps naming could be improved?